pkg/server/stream/v3/stream.go

```diff
@@ -26,7 +26,8 @@ type StreamState struct {
 	// Indicates whether the original DeltaRequest was a wildcard LDS/RDS request.
 	IsWildcard bool
 
-	// ResourceVersions contain a hash of the resource as a version map.
+	// ResourceVersions contain a hash of the resource as the value of the map.
+	// The name is the version key.
```

Comment:
This seems a bit confusing, this should probably speak about resource names somewhere?

Reply:
Maybe this is a little clearer?
```go
	// ResourceVersions contains a hash of the resource as the value and the resource name as the key.
	// This field stores the last state sent to the client.
```

Reply:
Yep sounds better